test/asynchronous/test_cursor.py

```diff
@@ -1444,6 +1455,21 @@ async def test_command_cursor_to_list_length(self):
         result = await db.test.aggregate([pipeline])
         self.assertEqual(len(await result.to_list(1)), 1)
 
+    @async_client_context.require_change_streams
+    @async_client_context.require_failCommand_fail_point
+    async def test_command_cursor_to_list_csot_applied(self):
```

Comment:
2 things:
- Let's use regular aggregate here since there's nothing special about $changeStream here and it complicates the test.
- The failCommands needs to be for "getMore", not "aggregate" since `await client.db.test.aggregate()` actually runs the aggregate command and to_list only runs getMore. We'll need to set batchSize to ensure at least 1 getMore even runs.